src/eureka/lib/meanerr.py

```diff
@@ -0,0 +1,114 @@
+# $Author: carthik $
+# $Revision: 267 $
+# $Date: 2010-06-08 22:33:22 -0400 (Tue, 08 Jun 2010) $
+# $HeadURL: file:///home/esp01/svn/code/python/branches/patricio/photpipe/lib/meanerr.py $
+# $Id: meanerr.py 267 2010-06-09 02:33:22Z carthik $
+
+import numpy as np
+
+
+def meanerr(data, derr, mask=None, err=False, status=False):
+    """
+    Calculate the error-weighted mean and the error in the
+    error-weighted mean of the input data, omitting masked data, NaN
+    data or errors, and data whose errors are zero.
+
+    Parameters:
+    -----------
```

Comment:
No colon after Parameters and Returns or other section titles (and then shorten the underlines by one)